src/routes/rss.xml.ts

```diff
@@ -0,0 +1,41 @@
+import rss from "@astrojs/rss";
+import { SITE_TITLE, SITE_DESCRIPTION } from "@src/consts";
```

Comment:
unused (we probably need to add a linter at some point to find all unused imports and have consistent styling)